src/booking/BookingCalendar.tsx

```diff
@@ -88,27 +100,15 @@ export const BookingCalendar: FunctionComponent<BookingCalendarProps> = ({
               >
                 <div>
                   <h3>
-                    {parse(
-                      dateLocationsPair.dateStr,
-                      "yyyy-MM-dd",
-                      new Date()
-                    ).toLocaleDateString([], {
+                    {
+                      new Date(dateLocationsPair.dateStr).toLocaleDateString([i18next.language], {
```

Comment:
@jimmymasaru I see you removed the date parsing. It was added so that we could define the format of the source date string when we construct a new Date() - such as avoiding confusion with 10/01/2021 being read as 01/10/2021. What was your rationale for removing it? Wouldn't it make the code more robust if we keep the parse to format but more it inside on `new Date([here])`?